test/EFCore.PG.FunctionalTests/BulkUpdates/NorthwindBulkUpdatesNpgsqlTest.cs

```diff
@@ -1121,6 +1210,72 @@ public override async Task Update_with_outer_apply_set_constant(bool async)
 WHERE c.""CustomerID"" = t0.""CustomerID""");
     }
 
+    public override async Task Update_with_cross_join_left_join_set_constant(bool async)
```

Comment:
Fails with invalid reference to FROM-clause entry for table "c"

Reply:
So this does pass for Sqlite fine. (I guess Sqlite decided to defer from npgsql here). So looks like npgsql cannot use the table being updated in anywhere except for where predicate. This test is using it in join predicate for left join. This is something you may need to handle in provider given difference from Sqlite.

Reply:
OK @smitpatel, opened #2478 to track doing this on my side (will ping you when I get to it), and approved https://github.com/dotnet/efcore/pull/28834. Thanks for looking at this.